src/System.Device.Gpio/System/Device/Gpio/Drivers/OrangePiLite2Driver.Linux.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.Gpio.Drivers
```

Comment:
We can create devices/Drivers directory for all of the community driven drivers - we'll eventually try to catch up with our lab and move them to System.Device.Gpio though